Vagrantfile

```diff
@@ -0,0 +1,38 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+
+  config.vm.provision "shell", :privileged => false, inline: <<-SHELL
+    sudo apt-get update
+
+    sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+    git clone https://github.com/pyenv/pyenv.git /home/vagrant/.pyenv
+    echo 'export PYENV_ROOT="$HOME/.pyenv"' >> /home/vagrant/.profile
+    echo 'export PATH="$PYENV_ROOT/bin:$PATH"' >> /home/vagrant/.profile
+    echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> /home/vagrant/.profile
+
+    exec "$SHELL"
```

Comment:
I don't think this does anything here. You can do it to start a new shell for subsequent commands but this is the end of the current script.

The second provisioning script will run regardless.

But also there's no need to have two separate scripts. Just put them together with `source ~/.profile` in between to run your additions to the .profile file.